plugins/faustwp/includes/rest/callbacks.php

```diff
@@ -258,8 +258,13 @@ function handle_rest_process_telemetry_callback( \WP_REST_Request $request ) {
 		'multisite'                                    => is_multisite(),
 		'php_version'                                  => PHP_VERSION,
 		'wp_version'                                   => get_wp_version(),
+		'engagement_time_msec'                         => 100,
+		'session_id'                                   => md5( get_telemetry_client_id() ),
```

Comment:
From GA docs:

> In order for user activity to display in reports like [Realtime](https://developers.google.com/analytics/devguides/reporting/data/v1/realtime-basics), engagement_time_msec and session_id must be supplied as part of the params for an event.

Due to delays in updating events in the GA UI, having 'realtime' work was the only reliable way I could determine that "it worked" without waiting an arbitrary/unknown amount of time for the Events page to show.

The way this `session_id` is set here, it would always be the same (and still unique), but maybe we want a way to roll it. From GA docs:

> In Analytics, a session initiates when a user either opens your app in the foreground or views a page or screen and no session is currently active (e.g. their previous session has timed out).

> By default, a session ends (times out) after 30 minutes of user inactivity. There is no limit to how long a session can last.

Thoughts?

Reply:
I don't see any downsides in having an indefinite session ID based on the user's unique telemetry ID. We are not tracking bounce rate or other time based things since we are not a traditional website, so from my perspective I don't think it matters if we roll it or not.

Reply:
Good points. Sounds good to me.